### ISSUE TITLE
setLinesPerRangeLabel() Doesn't Do What It Says

### ISSUE THREAD

**Issue by KennethEvans** [halfhp/androidplot#114]
* Description of the problem: setLinesPerRangeLabel(n) has no effect on the labeling. There is one label per range step not one label per n range steps.
* Steps to reproduce the issue: See below
* Version(s) of Androidplot being used:  1.5.9
* A stacktrace if the issue is causing a crash:  No crash

It is labeling every minor grid line, not every major grid line. It does indicate the major and minor grid lines. Such labeling, of course, is not what you want.

The screen dump is the result of this code:

```
        XYSeriesFormatter<XYRegionFormatter> formatter =
                new LineAndPointFormatter(Color.RED, null, null, null);
        formatter.setLegendIconEnabled(false);
        mSeries = new SimpleXYSeries(SimpleXYSeries.ArrayFormat.Y_VALS_ONLY,
                "Series 1", 0, 10, 20, 30, 40, 50, 60, 70, 80, 90);
        mPlot.addSeries(mSeries, formatter);
        mPlot.setRangeStep(StepMode.INCREMENT_BY_VAL, 1);
        mPlot.setLinesPerRangeLabel(10);
```

![Screenshot_Androidplot_Test small](https://user-images.githubusercontent.com/19721186/149601250-c74cf4e5-c417-4593-ab18-e6ae4f68d41f.jpg)

**Comment by KennethEvans** [comment]
I agree the response has been underwhelming. It should not be hard to fix. What are you planning to use?

**Comment by KennethEvans** [comment]
It's a kludge, but it looks like it would work. In my case I already decided to just not use y labels. I will keep this in mind. Thanks.

**Comment by halfhp** [comment]
fixed in 1.5.10
